snac-cooperative/snac-laravel
# [#34653] Relate and Unrelate Vocabulary Concepts to other concepts

**Link to Ticket:**  
https://app.shortcut.com/snac/story/34653/relate-and-unrelate-vocabulary-concepts-to-other-concepts

**PR Summary:**  
Fixes up concept relationships logic, adds relationship api routes, and adds relationship management to the concept edit interface.

**Testing Instructions:**  
1. Confirm that a vocabulary editor can now add and remove concept relationships when editing a concept.
2. Tests pass when running `ddev artisan test`

**Notes:**
- There are some vestigial bits in the blade views, vue components, and controllers that I ran out of time before being able to remove which should be addressed in an ideal world.
